packages/dev-server/src/HermesBundler.ts

```diff
@@ -0,0 +1,154 @@
+import type { Platform } from '@expo/config';
+import spawnAsync from '@expo/spawn-async';
+import fs from 'fs-extra';
+import type { composeSourceMaps } from 'metro-source-map';
+import os from 'os';
+import path from 'path';
+import process from 'process';
+import resolveFrom from 'resolve-from';
+
+export async function shouldBuildHermesBundleAsync(
+  projectRoot: string,
+  platform: Platform
+): Promise<boolean> {
+  if (platform === 'android') {
+    const gradlePropertiesPath = path.join(projectRoot, 'android', 'gradle.properties');
```

Comment:
Is this file path stable even with kotlin? I noticed a bunch of checks regarding this stuff here: https://github.com/expo/expo-cli/blob/master/packages/config-plugins/src/android/Paths.ts

Reply:
thinking so. it looks like that even from kotlin project, the file is still gradle.properties.